### DISCUSSION TITLE
git_files not used when in git folder

### DISCUSSION THREAD

**Discussion by Peter-Oestergaard** [LazyVim/LazyVim#4629] [Q&A] [answered]
Hi

Complete noob here. I am watching the Zero to IDE with LazyVim linked to from the LazyVim main page. About 3:20 in they talk about searching for files in git with `<space><f><f>`. When I do the same I can only use find_files.

I can find no mention of git_files anywhere in my nvim config folder.

What am I supposed to look for?

**Comment by dpetka2001** [comment, marked as answer]
Just use `<leader>fg` for `git_files`.

**Comment by Peter-Oestergaard** [comment]
Well, that was simple! Thanks.
